source/kubernetes/kubectl-plugin-reference.rst

```diff
@@ -0,0 +1,620 @@
+  plugin for each :mc-cmd:`volume <kubectl minio tenant create volumes>` 
+  the ``minio`` server uses. For example, if the MinIO deployment will have
+  4 servers with 4 volumes each, the Kubernetes cluster should have 16
+  total ``PV`` created using the MinIO Direct CSI plugin.
+
+- ``kubectl`` configured for access to the target Kubernetes cluster. Note
+  that ``kubectl`` is suppored within one minor version older or newer than
+  the ``kube-apiserver`` on the destination cluser. See
+  the Kubernetes documentation 
+  :kube-docs:`Kubernetes version and version skew support policy
+  <setup/release/version-skew-policy>`
+
+This procedure assumes the default ``kubectl`` context points to the
+Kubernetes cluster in which you want to deploy the MinIO tenant. To
+specify a different context, include the ``--context`` argument to
+``kubectl``. For more information on Kubernetes contexts, see
+:kube-docs:`Organizing Cluster Access using kubeconfig Files
+<concepts/configuration/organize-cluster-access-kubeconfig/>`
+
+1) Download the ``kubectl-minio`` Plugin
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The :minio-git:`Github Releases Page <opreator/releases>` lists all
+releases for the MinIO ``operator`` and the ``kubectl-minio`` plugin.
+
+Download the latest release onto a host machine with ``kubectl`` installed.
+The following command downloads the latest stable version of the 
+``kubectl-minio`` plugin:
+
+.. code-block:: shell
+   :class: copyable
+   :substitutions:
+
+   wget https://github.com/minio/operator/releases/download/|operator-release|/|kubectl-plugin-release| \
+      -O ~/Downloads/kubectl-minio
+
+You may need to use ``chmod`` or an equivalent utility to set the binary file
+as executable:
+
+.. code-block:: shell
+   
+   sudo chmod +x ~/Downloads/kubectl-minio
+
+2) Move the ``kubectl-minio`` Binary into your System Path
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The location of your system path depends on your filesystem.
+
+- For Linux and OSX, type ``echo $PATH`` into a terminal to check which
+  directories are included in the system path. Copy the ``kubectl-minio``
+  plugin into the appropraite directory. For example,
+  ``/usr/local/bin``.
+
+- For Windows, ensure the path to the ``kubectl-minio`` binary exists in the
+  system ``PATH`` environment variable. Defer to documentation on your version
+  of Windows for instructions on setting the ``PATH`` variable.
+
+3) Validate the Installation
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Run the following ``kubectl`` command to validate the installation:
+
+.. code-block:: shell
+
+   kubectl minio operator --version
+
+The operation should return the latest version of the ``kubectl-minio`` plugin.
+
+4) Create the Kubernetes Namespace
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Run the following ``kubectl`` command to create the namespace for the
+MinIO tenant:
+
+.. code-block:: shell
+   :class: copyable
+
+   kubectl create ns MinIOTenant1
```

Comment:
It is customary to have resources like namespace and secrets in lower-case, e.g. `minio-tenant1`